### PR TITLE
Adding how-to's for HA builder, basic cookstyle, and A2 Apps Access

### PR DESCRIPTION
**Pull request by scottvidmar**

### CODE REVIEW

**Comment by danielcbright** [comment]
hey @scottvidmar thanks for the PR! can you update the main README with links to your docs?

**Comment by scottvidmar** [comment]
Done!

**Comment by scottvidmar** [comment]
tas50 let me know that the cookstyle auto-remediation will change soon, but this should work for now. We'll need to update that once we pull in the Rubocop changes causing this.

File: examples/A2GrantAppDashboardAccess.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "projects": [
+        "*"
+      ]
+    }
+  ]
+}
+```
+2. POST the policy. I used an admin token, but any token that can manage IAM policies should work:
+```
+export TOKEN=`chef-automate iam token create admoon --admin`
```
Comment (jmassardo) [inline examples/A2GrantAppDashboardAccess.md:41]: @scottvidmar Looks like a typo in the command `admoon`?

File: examples/HabitatBuilderWithS3RDS.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Setup On-Premise Depot to Leverage RDS and S3
+
+1. Download the Zip archive of the on-prem-builder repo:<br />
+`curl -LO https://github.com/habitat-sh/on-prem-builder/archive/master.zip`
+2. Download the Chef Habitat cli tool:<br />
+`curl -Lo hab.tar.gz https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz`
+3. From the zip archive, install the hab binary somewhere in $PATH and ensure it has execute permissions:<br />
+`sudo chmod 755 /usr/bin/hab`<br />
+`sudo hab # read the license and accept if in agreement, as the root user`<br />
```
Comment (jmassardo) [inline examples/HabitatBuilderWithS3RDS.md:28]: Consider changing this to `hab license accept` to aide scriptability

File: examples/A2GrantAppDashboardAccess.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+curl -X POST https://localhost/apis/iam/v2/policies --data-binary @./policy -k -H "api-token: $TOKEN"  -v
+```
+3. Double check in the interface to ensure that all members been added. In the GUI: Settings -> Policies -> {Name of your policy, my example is “applications-viewer-access} -> Members
```
Comment (jmassardo) [inline examples/A2GrantAppDashboardAccess.md:44]: Minor grammar: `ensure that all members been added.`

Consider `ensure that all members exist in policy.` or similar

File: examples/WorkstationWriteBasicCookstyle.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
+# How-to - Write a basic Cookstyle Rule
```
Comment (jmassardo) [inline examples/WorkstationWriteBasicCookstyle.md:1]: This is a great start! Would it be possible to add a working example with some details about each section of it?

Comment (scottvidmar) [reply examples/WorkstationWriteBasicCookstyle.md:1]: It is. The document is going to need to change, anyhow, and add how to do these things in the latest Rubocop versions.

File: examples/A2GrantAppDashboardAccess.md (hunk 1 above)
Comment (scottvidmar) [reply examples/A2GrantAppDashboardAccess.md:41]: Not a typo, but I'm changing it to be more clear that this is the token's name being created.

File: examples/WorkstationWriteBasicCookstyle.md (hunk 4 above)
Comment (scottvidmar) [reply examples/WorkstationWriteBasicCookstyle.md:1]: done!

**Comment by scottvidmar** [comment]
Updated to address changes

File: examples/A2GrantAppDashboardAccess.md (hunk 3 above)
Comment (scottvidmar) [reply examples/A2GrantAppDashboardAccess.md]: Fixed.

File: examples/HabitatBuilderWithS3RDS.md (hunk 2 above)
Comment (scottvidmar) [reply examples/HabitatBuilderWithS3RDS.md]: Fixed. Good call.

File: examples/WorkstationWriteBasicCookstyle.md (hunk 4 above)
Comment (danielcbright) [reply examples/WorkstationWriteBasicCookstyle.md:1]: thanks!

**Comment by danielcbright** [comment]
thanks @scottvidmar !!!

**Pull request merged**